buildprocess/webpack.config.make.js

```diff
@@ -29,7 +29,7 @@ module.exports = function(hot, dev) {
                 {
                     // Don't let jasmine-ajax detect require and import jasmine-core, because we bring
                     // in Jasmine via a script tag instead.
-                    test: require.resolve('terriajs-jasmine-ajax'),
+                    test: require.resolve('jasmine-ajax'),
```

Comment:
Hey @na9da. Can you explain changing terriajs-jasmine-ajax to jasmine-ajax?

Reply:
Is it one/a mix of:
* Everything we needed in `jasmine-ajax` got merged upstream
* We no longer need the things we did need to make the custom package for because we're not using Browserify any more

Reply:
My initial reasoning was that our fork is 4 years old and hey this change didn't break any tests so lets ship it :smile: I was also looking to use a feature that didn't exist in our fork.

Then I found [this PR](https://github.com/jasmine/jasmine-ajax/pull/150) from Kevin. I think the code he is trying to fix is no longer present in [upstream](https://github.com/jasmine/jasmine-ajax/blob/master/src/boot.js#L34). We are also not using browserify any more. So I guess it ticks both of your rationale.